game_event/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from game_event_player.models import GameEventPlayer
+from court.models import Court
+
+
+class BallGame(models.TextChoices):
+    Soccer = 'Soccer', 'Soccer'
+    Basketball = 'Basketball', 'Basketball'
+    Volleyball = 'Volleyball', 'Volleyball'
+    Baseball = 'Baseball', 'Baseball'
+    Tennis = 'Tennis', 'Tennis'
+    Rugby = 'Rugby', 'Rugby'
+    Golf = 'Golf', 'Golf'
+    Cricket = 'Cricket', 'Cricket'
+    Handball = 'Handball', 'Handball'
+
+
+class Rating(models.IntegerChoices):
+    One = 1
+    Two = 2
+    Three = 3
+    Four = 4
+    Five = 5
+    Six = 6
+    Seven = 7
+    Eight = 8
+    Nine = 9
+    Ten = 10
+
+
+class GameEvent(models.Model):
+    id = models.IntegerField(primary_key=True)
+    time = models.DateTimeField(null=False)
+    level_of_game = models.IntegerField(choices=Rating.choices, blank=True)
+    min_number_of_players = models.PositiveIntegerField()
+    max_number_of_players = models.PositiveIntegerField()
+    ball_responsible_player = models.ForeignKey(GameEventPlayer, on_delete=models.CASCADE)
+    court_id = models.ForeignKey(Court, on_delete=models.CASCADE)
+    ball_game = models.CharField(
+        max_length=100,
```

Comment:
small comment I think you can make the length lower here

Reply:
> small comment I think you can make the length lower here

done